src/todolist.js

```diff
@@ -17,9 +17,6 @@ const addItem = (task) => {
 
 const removeItem = (index) => {
   todoList.splice(index, 1);
-  todoList.forEach((item, i) => {
-    item.index = i + 1;
-  });
   localStorage.setItem('todoList', JSON.stringify(todoList));
 };
```

Comment:
- [ ] Good job working on this milestone. Adding and removing to-do items is part of what needs to have been completed before this milestone, but the removal does not quite work properly. Clicking on the delete icon of an item only ends up removing the first item on the list. 
- [ ] Also, removing items either by clicking on the delete icon or the clear all completed button does not update the index of the rest of the items as expected.
- [ ] Please fix all issues and request for another review